_extras/checkout.md

```diff
@@ -105,6 +102,21 @@ The lessons you may submit to are:
 3.  [Stable Software Carpentry lessons](https://software-carpentry.org/lessons/):
     we are currently accepting solutions to existing exercises, diagrams, and improvements to learning objectives.
 
+> ## What Makes a Good Exercise?
+>
+> 1.  Follow the coding style of the lesson you're submitting to.
+>     For example,
+>     [our SQL lesson][swc-sql] capitalizes keywords but uses lower case for table and field names.
+> 2.  Include a solution to your exercise.
+> 3.  Make sure the exercise runs on Windows, Mac OS X, and Linux,
+>     since most workshops have users on all three platforms.
```

Comment:
I don't understand the difference between targeting all three and making sure it runs on all three.